sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -360,6 +360,54 @@ class ExplainSuite extends ExplainSuiteHelper with DisableAdaptiveExecutionSuite
       }
     }
   }
+
+  test("Explain formatted output for scan operator for datasource V2") {
+    withTempDir { dir =>
+      Seq("parquet", "orc", "csv", "json").foreach { fmt =>
+        val basePath = dir.getCanonicalPath + "/" + fmt
+        val pushFilterMaps = Map (
+          "parquet" ->
+            "|PushedFilers: \\[.*\\(id\\), .*\\(value\\), .*\\(id,1\\), .*\\(value,2\\)\\]",
+          "orc" ->
+            "|PushedFilers: \\[.*\\(id\\), .*\\(value\\), .*\\(id,1\\), .*\\(value,2\\)\\]",
+          "csv" ->
+            "|PushedFilers: \\[IsNotNull\\(value\\), GreaterThan\\(value,2\\)\\]",
+          "json" ->
+            "|remove_marker"
```

Comment:
Can we simply put `""`?

Reply:
@dongjoon-hyun I had tried and it didn't work for me. Perhaps there is a better way to do this. Basically, for JSON, i don't want a line printed for pushedFilters. Putting a `""` results in the following as expected output. Here i wanted to get rid of the empty line between `PartitionFilters` and `ReadSchema`
```
\(1\) BatchScan
Output \[2\]: \[value#x, id#x\]
DataFilters: \[isnotnull\(value#x\), \(value#x > 2\)\]
Format: json
Location: InMemoryFileIndex\[.*\]
PartitionFilters: \[isnotnull\(id#x\), \(id#x > 1\)\]

ReadSchema: struct\<value:int\>
```